dev/docker_data/run.py

```diff
@@ -287,6 +287,11 @@ def load_cpac_data_config(data_config_file, participant_label,
                          'takes precedence over '
                          'maximum_memory_per_participant in the pipeline '
                          'configuration file.')
+parser.add_argument('--num_ants_threads', type=int, default=1,
```

Comment:
I think we want the default to be what's checked against https://github.com/FCP-INDI/C-PAC/blob/8dc756623d9f7853dd02c587c9eae24971aa6217/dev/docker_data/run.py#L563

Otherwise it'll never use the original value in `pipeline_setup.system_config.num_ants_threads` (right?)

If I'm thinking about this correctly, we probably want the default to be something < 1 (that won't really be the number of cores anyone would ever use) like https://github.com/FCP-INDI/C-PAC/blob/8dc756623d9f7853dd02c587c9eae24971aa6217/dev/docker_data/run.py#L271 or we could get fancy with `argparse.SUPPRESS` and `hasattr` instead of using an integer default

Reply:
set default to `default = 0`